managed/src/main/java/com/yugabyte/yw/scheduler/Scheduler.java

```diff
@@ -80,7 +80,7 @@ private void initialize() {
   /**
    * Iterates through all the schedule entries and runs the tasks that are due to be scheduled.
    */
-  private void scheduleRunner() {
```

Comment:
`visiblefortesting` not working with private so changed to public.

Reply:
VisibleForTesting is (mostly) documentation annotation - when you have to make things public just so that tests can see it you have to add that annotation. 
So you have to make it public and *ALSO* add that annotation.

Reply:
And you should keep it default (package) scope instead of public since your test is in the same package.

Reply:
I'm for the default (package) visibility too.